src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderItemsController.cs

```diff
@@ -118,5 +118,36 @@ public async Task<ActionResult<GetOrderItemModel>> GetAsync(CallOffId callOffId,
 
             return ValidationProblem(ModelState);
         }
+
+        [Authorize(Policy = PolicyName.CanManageOrders)]
+        [HttpDelete("{catalogueItemId}")]
+        public async Task<ActionResult> DeleteOrderItemAsync(
+            CallOffId callOffId,
+            CatalogueItemId catalogueItemId)
+        {
+            Expression<Func<Order, IEnumerable<OrderItem>>> orderItems = o =>
+                o.OrderItems.Where(i => i.CatalogueItem.Id == catalogueItemId);
+
+            var order = await context.Order
+                .Where(o => o.Id == callOffId.Id)
+                .Include(orderItems)
+                .ThenInclude(x => x.CatalogueItem)
```

Comment:
We generally avoid using `x` for single letter lambda parameter names. A letter that relates to the parameter is preferable, e.g. `i` might be better here.

Reply:
done